README.md

```diff
@@ -29,14 +29,28 @@ For detailed user documentation go to our documentation
 
 ## Supported Platforms
 
-The Cloud Tools for IntelliJ plugin primarily supports the following IDEs:
-
-* IntelliJ IDEA Community Edition 2017.1 or later
-* IntelliJ IDEA Ultimate Edition 2017.1 or later
-
-In addition to IntelliJ IDEA, the plugin provides limited support for other JetBrains IDEs:
-
-| | IntelliJ IDEA - Ultimate | IntelliJ IDEA - Community | PyCharm |
+The Cloud Tools for IntelliJ plugin primarily supports the following IDEs, version 2017.1 or later:
+
+* IntelliJ IDEA Community Edition 
+* IntelliJ IDEA Ultimate Edition
+
+In addition to IntelliJ IDEA, the plugin provides limited support for the following IDEs, version 2017.1 or later:
+* PyCharm Professional Edition
```

Comment:
just a single PyCharm entry is all we need. there is no difference in support between pro, community, and edu

Reply:
How about PyCharm (all editions) to emphasize that you can use the plugins for all 3 editions?

Reply:
can we just do PyCharm (Community and Professional)

to avoid taking up 2 lines?